storage/innobase/xtrabackup/src/xtrabackup.cc

```diff
@@ -2267,17 +2268,18 @@ static dberr_t get_id_from_dd_scan(const std::string &name, uint64 *id,
 
   ut_ad(schema_id != 0);
   if (schema_id == 0) {
-    msg("xtrabackup: can't find %s entry in mysql/schemata for tablespace %s\n",
-        db_name.c_str(), name.c_str());
+    xb::error() << "can't find " << db_name.c_str()
+                << " entry in mysql/schemata for tablespace " << name.c_str();
     return (DB_NOT_FOUND);
   }
 
   *id = dd_table_map[std::make_pair(schema_id, table_name)];
 
   ut_ad(*id != 0);
   if (id == 0) {
-    msg("xtrabackup: can't find %s entry in mysql/tables for tablespace %s\n",
-        table_name.c_str(), name.c_str());
+    xb::error() << "xtrabackup: can't find " << table_name.c_str()
+                << " entry in mysql/tables for tablespace ",
+        name.c_str();
```

Comment:
```suggestion
                << " entry in mysql/tables for tablespace " <<
        name.c_str();
```

Reply:
Good catch, I wonder why there is no compilation error